app/controllers/organizations_controller.rb

```diff
@@ -15,28 +15,28 @@ def edit
   def update
     if @organization.update(user_params)
       authorize @organization
-      redirect_to @organization
+      redirect_to @organization, notice: 'Organization was sucessfully updated'
     else
       render 'edit'
```

Comment:
If you render the ":edit" method, you need to define it in the Controller

Reply:
Is edit not defined right before it? Or is the render 'edit' syntax wrong?